elements/pl-checkbox/pl-checkbox.py

```diff
@@ -453,7 +462,8 @@ def test(element_html, data):
             data['raw_submitted_answers'][name] = correct_keys
         else:
             pass  # no raw_submitted_answer if no correct keys
-        data['partial_scores'][name] = {'score': 1, 'weight': weight}
+        feedback = [option['feedback'] for option in data['params'][name] if option['key'] in correct_keys]
```

Comment:
This shouldn't break in practice, but let's use `.get()` to be extra careful.

```suggestion
        feedback = [option.get('feedback', None) for option in data['params'][name] if option['key'] in correct_keys]
```